homeassistant/components/telegram_bot/__init__.py

```diff
@@ -153,6 +153,7 @@
         vol.Optional(ATTR_PASSWORD): cv.string,
         vol.Optional(ATTR_AUTHENTICATION): cv.string,
         vol.Optional(ATTR_VERIFY_SSL): cv.boolean,
+        vol.Optional(ATTR_TIMEOUT): cv.positive_int,
```

Comment:
Isn't this a duplicate? Since you add it to the `BASE_SERVICE_SCHEMA` before and now extend it further.

Reply:
Yes, you’re right, I need to make changes and create a new pull request?

Reply:
You should make these changes and then push them. No, you do not need to create a new pull request since this one will update automatically as soon as you push any changes.

Reply:
You should only add it to `BASE_SERVICE_SCHEMA` and remove this duplicate here.
Otherwise you will have to add it every single send service schema which would be redundant.

Reply:
Ok